### PR TITLE
Reorganise systemseed/falcon into a monorepo with backend and frontend packages

### PR DESCRIPTION
**Pull request by kalabro**

### CODE REVIEW

File: falcon/modules/custom/falcon_development/falcon_development.install [hunk 1]
```diff
@@ -13,13 +13,28 @@ use Drupal\user\Entity\User;
  */
 function falcon_development_install() {
   falcon_development_activate_test_users();
+
```
Comment (PavelBulat) [inline falcon/modules/custom/falcon_development/falcon_development.install]: Do we need it? or should I add consumer to default content and update env.default file ?

File: falcon/settings/development.services.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Local development services.
+#sdfsdf sf sdf
```
Comment (PavelBulat) [inline falcon/settings/development.services.yml]: Remove it.

File: falconjs/templates/default/server.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-console */
```
Comment (PavelBulat) [inline falconjs/templates/default/server.js:1]: The file will be rewritten after merge falcon routing. For now the file do nothing.

**Review by PavelBulat — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -28,4 +29,73 @@ the sources:
...
+We recommend to use a tool like [commitizen](https://github.com/commitizen/cz-cli) or a plugin like [Git Commit Template](https://plugins.jetbrains.com/plugin/9861-git-commit-template) for preparing informative commit messages.
+
+### Examples
+
+**Commit message that triggers a major release, i.e. 1.4.3 → 2.0.0**
+
+```
+feat(backend): change all API endpoints to use new format.
+
+BREAKING CHANGE: All API endpoints now use new output format.
```
Comment (PavelBulat) [inline README.md:80]: @kalabro Can use stress the importance of breaking change comment.

Comment (PavelBulat) [reply README.md:80]: Note that there is a comment that starts with **BREAKING CHANGE:** ! The rest of commit message does not matter.

File: falcon/settings/development.services.yml (hunk 2 above)
Comment (PavelBulat) [reply falcon/settings/development.services.yml]: done.

File: falcon/modules/custom/falcon_development/falcon_development.install (hunk 1 above)
Comment (PavelBulat) [reply falcon/modules/custom/falcon_development/falcon_development.install]: skip it

File: .circleci/config.yml [hunk 5]
```diff
@@ -32,34 +48,96 @@ jobs:
...
-          command: echo 127.0.0.1 falcon.docker.localhost | sudo tee -a /etc/hosts
+          command: echo 127.0.0.1 falcon.docker.localhost app.docker.localhost | sudo tee -a /etc/hosts
+
+#      - run:
+#          name: Fix CircleCI files permissons
+#          command: |
+#            sudo chown ubuntu:ubuntu -R *
+#            sudo chown ubuntu:ubuntu -R .[!.]*
+#
+#      - restore_cache:
```
Comment (PavelBulat) [inline .circleci/config.yml]: @spleshka I believe you will explain me what I did wrong here! :)

**Review by PavelBulat — APPROVED** [review]

File: .circleci/config.yml (hunk 5 above)
Comment (spleshka) [reply .circleci/config.yml]: @PavelBulat given that everything works fine - nothing ;)

**Pull request merged**